doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -210,8 +210,9 @@ Content to be developed
 The number of Compute nodes (for workloads) determines the load on the controller nodes and networking traffic and, hence, the number of controller nodes needed in the OpenStack cloud; the number of controller nodes required is determined on the load placed on these controller nodes and the need for High availability and quorum requires at least 3 instances of many of the services on these controller nodes.
 
 <a name="3.4.1"></a>
-### 3.4.1. Virtualisation (KVM/QEMU)
-Content to be developed
+### 3.4.1. Virtualisation
+Virtualisation is a technology that enables a guest Operating System (OS) to be abstracted from the underlying hardware and software. This allow to run multiple VIrtual Machines(VMs) on the same hardware. Each such VMs have their own OS and are isolated from each other - application running on one VM does not have the access to resources of another VM. Such virtualisation is supported by various hypervisors available as open-source (KVM, Xen etc.) as well as commercial (Hyper-V, Citrix XenServer etc.). Selecting a hypervisor depends on the workload needs and the features provided by various hypervisors as illustrated in Hypervisor [Feature Support Matrix](https://docs.openstack.org/nova/latest/user/support-matrix.html). Openstack (Nova) allows to use various hypervisors within a single installation by means of scheduler filters like ComputeFilter, ImagePropertiesFilter etc.
```

Comment:
@msidana some minor points.
The first sentence is a little confusing. Should this be simply, "Virtualisation technology abstracts the underlying physical hardware allowing multiple virtual servers (machines) to run on the physical server."

"Each such VMs have their own OS and are isolated from each other - application running on one VM does not have the access to resources of another VM. " Wouldn't this depend upon the type of hypervisor (full, para, etc.)

Typos:
VIrtual
OpenStack 
and in "Openstack (Nova) allows to use various ..." should this be "OpenStack (Nova) allows the use of various ..."

Reply:
``` "Virtualisation technology abstracts the underlying physical hardware allowing multiple virtual servers (machines) to run on the physical server."``` -> Virtualisation may not necessarily abstract **underlying physical hardware only**. For example, KVM+QEMU enables us to run VMs on VMs. Thus I mentioned ```... that enables a guest Operating System (OS) to be abstracted from the underlying hardware and software``` to keep it generic without going into specifics like type1, type2 hypervisor(s), para/full virtualisation etc.

Corrected typos.